java/src/main/java/com/training/db/hw2/Algorithm.java

```diff
@@ -0,0 +1,17 @@
+package com.training.db.hw2;
+
+public class Algorithm {
+
+    public static void main(String[] args) {
+
+        int n1 = UiUtil.input("Input first number: ");
+
+        if (n1 == 1) {
```

Comment:
In this case much better to use switch-case construction, because in case of new algo it will lead to massive if-else construction which will decrease readability of code.

Reply:
done